### ISSUE TITLE
Markdown inside nodes is not rendered properly

### ISSUE THREAD

**Issue by nmeise** [tomoyukim/vscode-mermaid-editor#86]
**Describe the bug**
In the preview, markdown code inside flowchart nodes does not render properly.

**To Reproduce**
I used the mermaid example code for markdown formatting inside flowcharts (https://mermaid.js.org/syntax/flowchart.html#markdown-formatting):
```
flowchart LR
    markdown["`This **is** _Markdown_`"]
    newLines["`Line1
    Line 2
    Line 3`"]
    markdown --> newLines
```

**Expected behavior**
According to the mermaid docs, this example should render like this:
![image](https://github.com/tomoyukim/vscode-mermaid-editor/assets/139361333/4469f0c3-11cc-4d61-845f-110e02b8e7dc)

**Screenshots**
Actually, in the preview it looks like this:
![image](https://github.com/tomoyukim/vscode-mermaid-editor/assets/139361333/e1e18749-de6e-454a-823c-4e1e5b8bca2f)

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.2 LTS 
 - VSCode 1.80.0
 - Extension version v0.18.1

**Additional context**

**Comment by tomoyukim** [comment]
Thank you for your report! I think this is caused by the old mermaid version. I published v0.19.1 with the latest mermaid.js. Could you try it?

**Comment by nmeise** [comment]
Yes, thank you, looks good now.
